### PR TITLE
Kick backend

### PR DESCRIPTION
**Pull request by opxz7148**
### Description

- Allow host to kick attendee from their activity.

### Functional Acceptance Criteria

- When login as host and PUT activites/<activity_id>/  with data {"attendee_to_remove": [<list of attendee id that going to got removed>]}. User that has id in list won't be a participant anymore. 
- Even list contain id of user that not a member of an activity, host of activity or invalid id only participant of activity that not a host will be remove.
- #180 
- #199

### Checklist

- [x] I have performed a self-review of my code.
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation, if applicable.
- [x] My code follows the Coding Guidelines.

### Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Documentation update
- [ ] Other (please specify)

### Additional Information

example request 
Let say activity id = 20 has host with id 1 and member with Id 2,3,4,5
PUT activities/20/
data = {
    ...
    "attendee_to_remove": [1, 2, 5, 99]
}
only user with id 2,5 will be removed from activity,

### CODE REVIEW

**Review by TeerapatTrepopsakulsin — APPROVED** [review]
Great work and refactoring. The kick system works perfectly fine. _Approve_

Some very minor suggestion:
- When kick/promote participants. it say "pending update", but didn't mention what I've clicked (maybe I misclick, I want to know)
- If I promote lots of participants and accidentally click "kick", I cannot change it, instead, I have to cancel and promote all the participants again.

File: activities/views/activity_detail.py [hunk 1]
```diff
@@ -59,26 +123,21 @@ def put(self, request: HttpRequest, *args: Any, **kwargs: Any) -> response.Respo
...
 
-        return response.Response(
-            {
-                "message": f"You have successfully edited the activity {res_dict.get('name')}",
-                "id": res_dict.get("id")
-            }
-        )
+        print(attendee_ids_to_remove)
+        attendee_to_remove.delete()
+        return
```
Comment (OmegaOoh) [inline activities/views/activity_detail.py:140]: It is pointless to have a return (which didn't return anything at the end of the method.

File: frontend/src/component/EditModal.vue
Comment (OmegaOoh) [inline frontend/src/component/EditModal.vue:1]: image
This is a minor issue, but I think you may confused about the quote usage.

**Review by OmegaOoh — CHANGES_REQUESTED** [review]
Overall is looking good, but I have some minor concerns. Please fix

**Pull request merged**